filebeat/tests/system/test_shutdown.py

```diff
@@ -63,9 +63,10 @@ def test_shutdown_wait_ok(self):
         assert len(registry) == 1
         assert registry[0]["offset"] == output["offset"]
 
+    @unittest.skip("Skipping unreliable test")
```

Comment:
Have had to skip the test, as it has been very unreliable and I didn't manage to reproduce it yet for investigation. Plus, this feature as is will be replaced with wait shutdown support directly provided by the new publisher pipeline.